templates/bridgeserver2.yaml

```diff
@@ -106,6 +106,8 @@ Parameters:
   NewRelicLicenseKey:
     Type: String
     NoEcho: true
+  ParticipantFileBucket:
```

Comment:
remove this

Reply:
removed (okay I get what it's doing now... it's setting an environment variable to the name of the bucket resource)